### ISSUE TITLE
prompt / interactive commands flood the screen repeatedly

### ISSUE THREAD

**Issue by nvgoldin** [gnunn1/tilix#1179]
Sometimes my prompt is being printed repeatedly (see screen shot). Also when I run interactive commands that refresh the screen ( such as top ), it just floods the screen - each update prints another line.
I'm using a combination of tilix + powerline. I suspect this is tilix, as I tried using the normal gnome terminal and it doesn't happen there. Afterwards I tried using the system-default .bashrc instead of mine and it still happens. I tried following https://github.com/gnunn1/tilix/wiki/VTE-Configuration-Issue and used "Run command as login shell" option to no avail.

Any ideas?

![image](https://user-images.githubusercontent.com/13018704/32433030-44724246-c2e2-11e7-9d40-38336e9bc498.png)

**Comment by nvgoldin** [comment]
Just noticed that it usually happens when changing the size of the window frequently. If I open a brand new tilix session(in the middle of the screen) and run 'top' sometimes it works as expected, if then I move to full screen mode(F11) and back, it starts flooding.

**Comment by gnunn1** [comment]
Does it work better if you disable the overlay scrollbar in Preferences | Appearance?

Also what DE are you using, gnome?

**Comment by egmontkob** [comment]
Same as #1122?

**Comment by nvgoldin** [comment]
> Does it work better if you disable the overlay scrollbar in Preferences | Appearance?

Yep! in initial inspection looks like the problem is gone.

> Same as #1122?

Sounds very similar.

Tilix version:
````
 ngoldin  ~  tilix --version
Versions
	Tilix version: 1.6.4
	VTE version: 0.46
	GTK Version: 3.22.11

Tilix Special Features
	Notifications enabled=0
	Triggers enabled=0
	Badges enabled=0
```
Debian 9 + Cinnamon

**Comment by gnunn1** [comment]
Glad that it resolved the issue. @nvgoldin does Cinnamon set the ```GTK_OVERLAY_SCROLLING=0``` environment like Mate does?

**Comment by nvgoldin** [comment]
Seems so:
```
 ngoldin  ~  export | grep GTK
declare -x GTK_MODULES="gail:atk-bridge"
declare -x GTK_OVERLAY_SCROLLING="0"

```

**Comment by gnunn1** [comment]
Thanks, this has been corrected in 1.7
